tests/test_memory_unit.py

```diff
@@ -0,0 +1,42 @@
+from uuid import uuid4
+from app.main import create_memory, get_memory, memories_store
+
+
+def test_create_memory():
```

Comment:
This is nice example for the unit test @adithya-234 . When the system grows it will have higher complexity too so it would be great to extract memories_store functionality into another class which you can test here separately. So `main.py` should for example create an instance of MemoryService class which is stored in `memory_service.py` or whatever fits you. Then you haeve separate tests for main.py and separate for memory_service.py